### PR TITLE
Added feature for showing completed task

### PR DESCRIPTION
**Pull request by swaindhruti**
Here is the SS of working feature
Screenshot 2023-10-16 171601

### CODE REVIEW

File: cmd/todo/main.go [hunk 1]
```diff
@@ -13,14 +13,16 @@ import (
 )
 
 const (
-	todoFile = "data/todos.mayur"
+	todoFile    = "data/todos.mayur"
```
Comment (sudo-rsingh) [inline cmd/todo/main.go]: why is there space between **todoFile** and **=**

**Review by sudo-rsingh — APPROVED** [review]
LGTM just remove that space and good to go

**Comment by swaindhruti** [comment]
> LGTM just remove that space and good to go

yup sir removed the space

File: README.md [hunk 2]
```diff
@@ -30,4 +30,5 @@ cd GoDoList
 ./todo -complete=1
 ./todo -del=1
 ./todo -list
+./todo -showcompleted
```
Comment (Meru45) [inline README.md]: The flag should be in Camel Case

**Review by Meru45 — CHANGES_REQUESTED** [review]

File: cmd/todo/main.go [hunk 3]
```diff
@@ -14,13 +14,15 @@ import (
 
 const (
 	todoFile = "data/todos.mayur"
+	historyFile = "data/history.mayur"
```
Comment (Meru45) [inline cmd/todo/main.go:17]: We have to add the completed tasks to the history file.

**Review by Meru45 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (swaindhruti) [reply README.md]: Done👍👍

File: cmd/todo/main.go (hunk 3 above)
Comment (swaindhruti) [reply cmd/todo/main.go:17]: Working on it.

File: cmd/todo/main.go [hunk 4]
```diff
@@ -14,13 +14,15 @@ import (
...
 )
 
 func main() {
 	add := flag.Bool("add", false, "add a new todo")
 	complete := flag.Int("complete", 0, "mark a todo completed")
 	del := flag.Int("del", 0, "delete a todo")
-	list := flag.Bool("list", true, "show the list of todos")
+	list := flag.Bool("list", false, "show the list of todos")
```
Comment (sudo-rsingh) [inline cmd/todo/main.go:24]: Explain why have you changed it from true to false.

File: cmd/todo/main.go [hunk 5]
```diff
@@ -49,13 +49,14 @@ func main() {
 		}
 		todos.Print()
 	case *complete > 0:
-		err := todos.Complete(*complete)
+		err := todos.Complete(*complete, historyFile)
```
Comment (Meru45) [inline cmd/todo/main.go:52]: This historyFile cannot be pushed into the Complete function like this, here store function should be used to store completed todos in the history file

File: todo.go [hunk 6]
```diff
@@ -41,7 +41,15 @@ func (t *Todos) Complete(index int) error {
 
 	ls[index-1].CompletedAt = time.Now()
 	ls[index-1].Done = true
-
```
Comment (Meru45) [inline todo.go:44]: There is no need to re-write the complete function

**Review by Meru45 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**